crates/solvers/src/tests/legacy/mod.rs

```diff
@@ -48,18 +50,13 @@ pub async fn setup(expectations: Vec<Expectation>) -> SocketAddr {
 #[derive(Debug, Clone)]
 struct State(Arc<Mutex<Vec<Expectation>>>);
 
-/// Creates a temporary file containing the config of the given solver.
-pub fn create_temp_config_file(solver_addr: &SocketAddr) -> tempfile::TempPath {
-    let base_url = format!("http://{solver_addr}/solve");
-    let config = format!(
+/// Creates a legacy solver configuration for the specified host.
+pub fn config(solver_addr: &SocketAddr) -> tests::Config {
+    tests::Config::String(format!(
         r"
 solver-name = 'legacy_solver'
-solve-endpoint = '{}'
+solve-endpoint = 'http://{solver_addr}/solve'
 chain-id = '1'
-",
-        base_url
-    );
-    let mut file = tempfile::NamedTempFile::new().unwrap();
-    file.write_all(config.as_bytes()).unwrap();
-    file.into_temp_path()
+        ",
```

Comment:
You nitted on this in my other PR. I don't care myself. Just wanted to point it out since you preferred it that way. Same thing applies in `crates/solvers/src/tests/balancer/mod.rs` as well.
```suggestion
",
```

Reply:
The formatting nit was to vertically align the opening and closing `"` characters in the string (which is how it should be here).